integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage40.java

```diff
@@ -624,6 +654,483 @@ void testWithAISourceModelHomeSetToNone() {
+    Path modelsPath2 = Paths.get(errorpathAIPath2.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2),
+        "Create directory failed");
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, MII_BASIC_WDT_MODEL_FILE),
+        Paths.get(modelsPath2.toString(), MII_BASIC_WDT_MODEL_FILE),
+        StandardCopyOption.REPLACE_EXISTING), "Copy files failed");
+
+    // create image with model and no wdt installation files
+    // create image1 with model and wdt installation files
+    createAuxiliaryImage(errorpathAIPath2.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(),
+        errorPathAuxiliaryImage2);
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", errorPathAuxiliaryImage2, DOMAIN_IMAGES_REPO);
+      dockerLoginAndPushImageToRegistry(errorPathAuxiliaryImage2);
+    }
+
+    // create domain custom resource using auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
+        domainUid2, errorPathAuxiliaryImage2);
+
+    Domain domainCR = createDomainResource40(domainUid2, errorpathDomainNamespace,
+        WEBLOGIC_IMAGE_TO_USE_IN_SPEC, adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, replicaCount, "cluster-1", auxiliaryImagePath,
+        errorPathAuxiliaryImage2);
+
+    // create domain and verify it is failed
+    logger.info("Creating domain {0} with auxiliary image {1} in namespace {2}",
+        domainUid2, errorPathAuxiliaryImage2, errorpathDomainNamespace);
+    assertDoesNotThrow(() -> createDomainCustomResource(domainCR), "createDomainCustomResource throws Exception");
+
+    // check the introspector pod log contains the expected error message
+    String expectedErrorMsg = "The domain resource 'spec.domainHomeSourceType' is 'FromModel'  and "
+        + "a WebLogic Deploy Tool (WDT) install is not located at  'spec.configuration.model.wdtInstallHome'  "
+        + "which is currently set to '/aux/weblogic-deploy'";
+
+    verifyIntrospectorPodLogContainsExpectedErrorMsg(domainUid2, errorpathDomainNamespace, expectedErrorMsg);
+
+    // check the domain event contains the expected error message
+    checkDomainEventContainsExpectedMsg(opNamespace, errorpathDomainNamespace, domainUid2, DOMAIN_FAILED,
+        "Warning", timestamp, expectedErrorMsg);
+
+    // check the operator pod log contains the expected error message
+    String operatorPodName =
+        assertDoesNotThrow(() -> getOperatorPodName(OPERATOR_RELEASE_NAME, opNamespace),
+            "Can't get operator pod's name");
+    checkPodLogContainsString(opNamespace, operatorPodName, expectedErrorMsg);
+
+    // check there are no admin server and managed server pods and services created
+    checkPodDoesNotExist(adminServerPodName, domainUid2, errorpathDomainNamespace);
```

Comment:
You might want to move this down after service not exist check

Reply:
done